package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tilesplash",
   "description": "A light and quick nodejs webserver for serving mapbox vector tiles (and topojson) from a postgis backend",
-  "version": "2.1.2",
+  "version": "2.2.0",
```

Comment:
I'd recommend bumping the major version number rather than the minor because the API has changed from connection strings to config objects.